prefect_gcp/credentials.py

```diff
@@ -72,6 +72,9 @@ class GcpCredentials:
         project: Name of the project to use.
     """
 
+    _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/4CD4wwbiIKPkZDt4U3TEuW/c112fe85653da054b6d5334ef662bec4/gcp.png?h=250"  # noqa
+    _block_type_name = "Google Cloud Platform Credentials"
+
     service_account_file: Optional[Union[str, Path]] = None
     service_account_info: Optional[Union[str, Dict[str, str]]] = None
```

Comment:
I think this type will work better for this use case. Users can input a string, but it will be converted to a dictionary by `pydantic`. Note this type needs to be imported from `pydantic`.
```suggestion
    service_account_info: Optional[Json] = None
```